1-run-automation-test.md

```diff
@@ -0,0 +1,126 @@
++ In the navigation bar at the top of the Kobiton website, select `Devices`.
+ 
+ ![alt text](./assets/devices.png )
+ 
+ + Hover over any device and click on the Automation settings button. A dialog box will then appear on screen.
+ 
+ ![alt text](./assets/click_auto.png )
+ 
+ + On the left hand side, you can select your preferred language, App Type...
+ 
+ ![alt text](./assets/automation.png ) 
+ **b. Run automation test**
++ Kobiton already provide some example for basic automation test. For samples of automation tests, go to https://github.com/kobiton/samples .
++ Choose a language for your test script, and decide whether you want to test on Android or IOS , and either do a web test or an app test. 
++ Make sure in the code you specify your Kobiton username, API key and replace the `desiredCaps`.
+
+Example:
+
+```javascript
+const username = process.env.KOBITON_USERNAME
+const apiKey = process.env.KOBITON_API_KEY
+ const kobitonServerConfig = {
+  protocol: 'https',
+  host: 'api-test.kobiton.com',
+  auth: `${username}:${apiKey}`
+}
+ var desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  noReset:            true,
+  fullReset:          false, 
+  captureScreenshots: true,
+  app:                {your_app_id}, 
+  udid:               {your_device_udid}
+}
+```
+
+## 4. Configure your TeamCity project for Kobiton testing
+You can attach an enviroment variable to TeamCity(Optional).
++ In your project in TeamCity, on the left side on the page, click on 'Parameters'. Then click buton `Add new parameter`. 
+
+![alt text](./assets/param.png )
+
++ Add your Kobiton username and API key.
+
+![alt text](./assets/param_name.png )
+
+![alt text](./assets/param_key.png )
+
+Now, we will add a build step to run automation test.
++ In your project in TeamCity, on the left side on the page, click on 'Build Step' to set build steps.
++ On the right side of the page,  for the 'Runner type' category, choose 'Command Line '.
++ Add command lines to your script content.
+
+Example: 
+```
+  cd javascript
+  npm install
+  npm run android-web-test
+```
+> Note:
+> + `npm install` is the command if you using NodeJS 
+> + `npm run android-web-test` is the command to run the test script
+
+![alt text](./assets/cmd.png )
++ Now, try a build on Teamity and check Kobiton cloud devices to see if a test session was created.
+
+## 5. Call Kobiton REST API to get session information
+
+Kobiton already provide an script using NodeJS to get session information. For samples of get session, go to https://github.com/kobiton/samples and select the folder named 'REST API'. You can downloads the reposity and run script to get session information and get session commands.
```

Comment:
*provided a script.
*examples of how to get session(s) information, get session commands (list all of the functions that the REST API folder provides, or instead just say that "examples of how to interact with session(s)").
*can download, not "downloads"
*repository, not "reposity".